controller/consumer.go

```diff
@@ -0,0 +1,88 @@
+package controller
+
+import (
+	"encoding/json"
+	"log"
+
+	"github.com/FakJeongTeeNhoi/report-system/model"
+	amqp "github.com/rabbitmq/amqp091-go"
+)
+
+func failOnError(err error, msg string) {
+	if err != nil {
+		log.Panicf("%s: %s", msg, err)
+	}
+}
+
+func StartConsumeDataFromQueue(Exchange string, key []string) {
+
+	conn, err := amqp.Dial("amqp://guest:guest@rabbitmq:5672/")
```

Comment:
The connection string should be in `.env` for security purpose.

Reply:
done